src/components/ThemeMode/ThemeMode.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from './themeMode.module.scss';
+import { useRef } from 'react';
+import { getIsDarkMode, toggleDarkMode } from '../../config/themeMode/themeModeStorage';
+import IconButton from '../IconButton/IconButton';
+import clsx from 'clsx';
+import { FiSun, FiMoon } from 'react-icons/fi';
+
+const ThemeMode = () => {
```

Comment:
This is Button or Section underneath - I would call it ThemeButton / ThemeSwitcher / ThemeModeSwither /ThemeSelector - anything that sugggest action